easybuild/easyconfigs/g/gmsh/gmsh-4.7.1-foss-2020a-Python-3.8.2.eb

```
@@ -0,0 +1,40 @@
+easyblock = 'CMakeMake'
+
+name = 'gmsh'
+version = '4.7.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://gmsh.info/'
+description = """Gmsh is a 3D finite element grid generator with a build-in CAD engine and post-processor."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
+toolchainopts = {"usempi": True}
+
+source_urls = ['https://gmsh.info/src/']
+sources = ['%(name)s-%(version)s-source.tgz']
+checksums = ['c984c295116c757ed165d77149bd5fdd1068cbd7835e9bcd077358b503891c6a']
+
+builddependencies = [
+    ('CMake', '3.16.4'),
+    ('SWIG', '4.0.1'),
+]
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('PETSc', '3.12.4', versionsuffix),
+    ('SLEPc', '3.12.2', versionsuffix),
+]
+
+separate_build_dir = True
+
+configopts = '-DENABLE_FLTK=0 -DENABLE_WRAP_PYTHON=ON -DENABLE_METIS=1 '
+configopts += '-DENABLE_BUILD_LIB=ON -DENABLE_BUILD_SHARED=ON -DENABLE_BUILD_DYNAMIC=ON'
+
+modextrapaths = {'PYTHONPATH': ['lib64']}
+
+sanity_check_paths = {
+    'files': ['bin/gmsh', 'lib64/gmsh.py'],
```

@besserox Should we enhance this a bit, now that the shared library is also built?

```suggestion
    'files': ['bin/gmsh', 'lib64/gmsh.py', 'lib64/libgmsh.a', 'lib64/libgmsh.%s' % SHLIB_EXT],
```

---

pushed the correction for foss and intel